<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowNodesDockWidget</class>
 <widget class="QDockWidget" name="ShowNodesDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show entities by type</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QCheckBox" name="propertyCB">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Properties</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="individualCB">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>93</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Individuals</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="classCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>73</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Classes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
